### PR TITLE
update tRO mapLogin

### PR DESCRIPTION
**Pull request by sctnightcore**

### CODE REVIEW

**Comment by ya4ept** [comment]
1) these are the wrong packages, they don't exist:
```
001E 30 0 259
8B00000A -1 4 1
```
2) see your shuffle.txt

this is normal?
```
0089 0437
0437 035F
```

these packages don't make any sense:
```
0360 0360
0361 0361
0362 0362
0363 0363
0364 0364
0365 0365
0366 0366
0367 0367
0368 0368
0369 0369
...
0838 0838
0835 0835
0819 0819
0817 0817
0815 0815
0811 0811
0802 0802
...
07E4 07E4
0436 0436
02C4 02C4
...
0202 0202
022D 022D
023B 023B
```

File: tables/tRO/recvpackets.txt [hunk 1]
```diff
@@ -1,1582 +1,1584 @@
...
+0BA4 85 85 1
+0BA5 12 12 1
+0BA6 -1 8 1
+0BA7 -1 8 1
+0BA8 7 7 1
+0BA9 -1 8 1
+0BAA 22 22 1
+0BAB 22 22 1
+0BAC 22 22 0
+8B00000A -1 4 1
```
Comment (alisonrag) [inline tables/tRO/recvpackets.txt:1584]: to merge just need to fix this packet

**Review by alisonrag — CHANGES_REQUESTED** [review]

File: tables/tRO/recvpackets.txt (hunk 1 above)
Comment (ya4ept) [reply tables/tRO/recvpackets.txt:1584]: @sctnightcore

Comment (ya4ept) [reply tables/tRO/recvpackets.txt]: done: 38cf8593230a9e504c296ad2cd13ff1d56c8e630

**Review by ya4ept — APPROVED** [review]

**Comment by ya4ept** [comment]
ready to merge

**Review by alisonrag — APPROVED** [review]

**Pull request merged**